### ISSUE TITLE
Permanent notifications until a daily review is done

### ISSUE THREAD

**Issue by edvein-rin** [ankidroid/Anki-Android#8128]
**Is your feature request related to a problem? Please describe.**
It's happens that people close their notifications by a mistake (or is it only me?). So it's easy to forget about Anki's daily review if there is no reminder left.

**Describe the solution you'd like**
It would be nice to have an option (a checkbox in settings) to make a reminder permanent as a notification until a daily review is done. Then it just should destroy itself. Also it will be comfortable if this notification's counter of cards to review updates with new reviews done. So it would be possible to know how many cards are left just by looking at the notification after an interruption of reviews for any reason.

**Additional context**
I think there is no need in an additional UI update for  the notification but a dynamic cards to review counter.

**Comment by danishjamal104** [comment]
> **Is your feature request related to a problem? Please describe.**
> It's happens that people close their notifications by a mistake (or is it only me?). So it's easy to forget about Anki's daily review if there is no reminder left.
> 
> **Describe the solution you'd like**
> It would be nice to have an option (a checkbox in settings) to make a reminder permanent as a notification until a daily review is done. Then it just should destroy itself. Also it will be comfortable if this notification's counter of cards to review updates with new reviews done. So it would be possible to know how many cards are left just by looking at the notification after an interruption of reviews for any reason.
> 
> **Additional context**
> I think there is no need in an additional UI update for the notification but a dynamic cards to review counter.

Don't you think persistence notification won't have much effect as it is not a reminder but it stays their for the whole, where as reminder are meant for creating alert, rather what we can do is increase the frequency of reminders.

**Comment by edvein-rin** [comment]
I agree with you, @danishjamal104. It would be much more noticeable.
Then to summarize issues' point:
1. Add an option to choose a frequency of reminders.
2. New notifications from reminders should contain an updated information about a number of cards to review (probably already existing issue).

**Comment by haran2248** [comment]
@Arthur-Milchior @david-allison-1 Can you guys tell whether this issue is relevant?If yes can I work on this?

**Comment by Arthur-Milchior** [comment]
Updating/destroying the notifications are clear win, as long as it does not require to much background process.
The main risk is that notification process need database access, so it risks to slow down ankidroid while the new computation is done. Not something we can do regularly. 

We usually do not like adding to much options. But "reminder" is already a page by itself and is mostly empty, so I guess it would be okay to add a "+" button to add notification at multiple hours. I guess an extra button to put the notification permanent would be okay, assuming it is even possible. I mean, we already have a window mostly empty, so adding a button here does not really make the window less easy to read

**Comment by david-allison-1** [comment]
@haran2248 It'd be great to have someone work on this, but it'd be best to consolidate discussion on #4944. It feels that we have a lot of similar issues stemming from the need for a redesign, and keeping the discussion centralised would be useful.

**Comment by KonstantinDjairo** [comment]
how's the current state of this?

**Comment by david-allison** [comment]
AFAIK Still an issue

**Comment by KonstantinDjairo** [comment]
> AFAIK Still an issue

wish we had at least an option to pin the notification , so that it can't be "cleaned" , like a persistent notification
